api/src/main/java/org/openmrs/module/cohort/CohortMemberAttributeType.java

```diff
@@ -19,5 +19,15 @@
 
     @Getter
     @Setter
-    private Integer id;
+    private Integer cohortMemberAttributeTypeId;
+
+    @Override
+    public Integer getId() {
```

Comment:
does the getter and setter have to be of Id not of  cohortMemberAttributeTypeId?